maquetado/scss/margins.scss

```diff
@@ -0,0 +1,5 @@
+$book-element-margin-top: 5%;
+$element-text-margin-left: 1%;
+$book-image-margin: 1%;
+$book-title-margin-top: $book-image-margin*5;
```

Comment:
Avoid `margin-top` you can get the same result without doing this. Use pixels instead.